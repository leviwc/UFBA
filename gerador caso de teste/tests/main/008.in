28 319
4 14
16 26
17 12
9 19
21 24
9 20
1 7
15 6
12 4
28 7
3 28
21 17
22 1
4 5
20 14
24 14
16 23
13 28
11 6
13 14
27 13
25 28
18 1
3 2
7 22
3 5
4 3
8 26
8 12
18 17
2 17
11 7
8 4
20 2
9 2
20 1
17 24
18 26
4 21
12 3
28 12
4 25
13 1
21 28
21 27
22 13
21 16
3 16
17 19
14 14
25 23
24 7
5 2
9 13
19 28
4 1
25 11
24 22
9 21
3 22
9 17
26 21
11 16
25 22
23 12
1 8
3 1
25 20
26 28
18 15
13 19
13 8
11 12
6 28
7 19
9 22
21 10
15 27
9 6
7 15
12 15
22 4
16 13
1 6
25 27
20 22
17 25
25 1
16 16
12 16
26 1
15 23
1 28
10 7
16 2
6 8
6 7
13 23
4 19
3 3
27 28
26 19
4 6
7 13
17 3
11 21
24 28
1 23
12 5
8 14
26 24
25 26
12 27
17 16
13 21
10 27
20 10
14 2
2 27
27 1
25 8
9 26
11 3
4 16
11 1
11 15
2 26
14 15
1 17
22 19
3 27
24 16
18 28
12 18
1 15
5 8
24 25
11 28
10 6
18 19
9 16
16 27
23 21
18 14
6 22
23 10
9 25
7 5
12 9
20 5
3 26
22 17
18 20
7 14
25 13
2 12
3 23
14 23
19 5
24 5
25 25
24 20
8 15
1 2
1 24
18 16
9 10
23 17
6 16
3 15
21 2
18 18
17 14
25 3
9 24
16 28
25 5
2 4
6 12
17 7
9 8
23 20
24 2
20 27
4 24
18 22
11 23
4 10
1 10
14 26
24 6
21 1
3 20
2 11
22 28
28 10
28 20
27 5
5 11
10 17
12 20
23 6
16 14
23 5
5 21
8 3
10 8
16 10
2 7
9 27
10 24
27 19
10 5
11 18
20 8
4 4
1 14
3 9
7 16
11 9
3 13
27 4
19 16
10 22
5 22
28 17
25 7
8 11
22 21
19 15
8 18
24 3
15 28
2 19
21 3
13 18
11 27
20 6
16 22
25 18
2 22
28 2
12 7
23 23
20 11
5 28
23 9
6 5
8 27
15 15
25 21
25 14
7 4
17 6
3 14
15 13
2 10
9 4
11 22
24 27
12 12
10 25
26 17
15 9
9 7
16 15
26 27
13 17
8 28
20 4
26 12
25 6
11 24
24 23
17 27
12 21
14 27
23 18
9 5
15 21
26 6
22 26
18 3
27 6
2 6
19 12
11 11
13 20
8 2
5 14
4 18
14 19
19 24
15 10
19 20
23 27
24 8
27 18
21 6
25 12
7 23
8 21
28 4
8 16
19 3
13 13
2 15
12 22
23 4
7 26
2 25
2 13
12 13
2 18
8 19
23 19
5 16
23 28
15 20
