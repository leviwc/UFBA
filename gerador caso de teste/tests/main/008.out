Resultado com busca binaria: 4
1608.448000 s
Resultado com backtracking apenas: 4
0.041000 s
Resultado testando todas as possibilidades: 4
8827.445000 s
Resultado com busca binaria otimizada: 4
52.704000 s
